doc/install/install.md

```diff
@@ -1,26 +1,25 @@
 ## Installing & Deploying
-Installing and running the Deep Log Inspection system requires [git][1] and [docker][2], so make sure they are installed on your system before proceeding. In order for Elasticsearch to boot correctly, user limits must be set via [Docker configuration][5].
+Installing and running the Deep Log Inspection system requires [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Docker Engine](https://www.docker.com/) in [Swarm mode](https://docs.docker.com/engine/swarm/):
 
-Open your system's command line and `cd` to the folder where the project will be stored. Then, download the project by cloning the git repository:
+1. Set up the [Docker Swarm cluster](https://docs.docker.com/get-started/part4/#create-a-cluster) where services will be deployed
+2. Download the project by cloning the git repository:
 
-    git clone https://github.com/martel-innovate/deep-log-inspection.git
+        git clone https://github.com/martel-innovate/deep-log-inspection.git
 
-To install and run the system, also refer to the [Quick Start guide](../quickstart.md).
+3. Set user limits in Docker [configuration file](https://github.com/martel-innovate/deep-log-inspection/blob/master/log-server/config/docker.service). For more details, refer to the [Docker configuration section](docker.md)
+4. Create the overlay network where the services will be deployed, and the storage volumes for Elasticsearch data and logs:
 
-First, create the overlay network where the services will be deployed, and the storage volumes for Elasticsearch data and logs:
+        docker network create -d overlay backend
+        docker volume create esdata
+        docker volume create eslogs
 
-    docker network create -d overlay backend
-    docker volume create esdata
-    docker volume create eslogs
+5. Let `deeplog` be the name of the stack where we deploy all services to, then:
 
-Deploy the system with [Docker Swarm][3]:
+        docker stack deploy -c docker-compose.yml deeplog
 
-    docker stack deploy -c docker-compose.yml deeplog
-
-In development/test environment, you might want to start also keystone:
-
-    docker stack deploy -c docker-compose-keystone.yml deeplog
+The compose file can be found [here](https://github.com/martel-innovate/deep-log-inspection/blob/master/log-server/docker-compose.yml).
 
```

Comment:
How to verify that the system is up?

curl elasticsearch health api (return the status of the cluster)